src/lib/utils/print_utils.cpp

```diff
@@ -126,14 +140,42 @@ void print_table_key_constraints(const std::shared_ptr<const Table>& table, std:
   }
 }
 
-// We explicitly instantiate these template functions because clang-12(+) does not instantiate them for us.
-template void print_directed_acyclic_graph<const AbstractLQPNode>(
-    const std::shared_ptr<const AbstractLQPNode>& node, const NodeGetChildrenFn<const AbstractLQPNode>& get_children_fn,
-    const NodePrintFn<const AbstractLQPNode>& print_node_fn, std::ostream& stream);
-template void print_directed_acyclic_graph<const AbstractOperator>(
-    const std::shared_ptr<const AbstractOperator>& node,
-    const NodeGetChildrenFn<const AbstractOperator>& get_children_fn,
-    const NodePrintFn<const AbstractOperator>& print_node_fn, std::ostream& stream);
+void print_expressions(const ExpressionUnorderedSet& expressions, std::ostream& stream, const std::string& separator) {
+  auto expressions_vector = std::vector<std::shared_ptr<AbstractExpression>>{expressions.begin(), expressions.end()};
+
+  // Obtain first ColumnID found in expression.
+  const auto original_column_id = [](const auto& expression) {
+    // Initialize with maximum value. Thus, we print expressions where we cannot get the original ColumnID last.
```

Comment:
Don't get this comment.

Reply:
`value_(1)` does not have an original ColumnID from an `LQPColumnExpression`. Thus, the ColumnID here stays in the initialized state and the expression would end up at the end.

Reply:
`    // Initialize with maximum value to print expressions where we cannot get the original ColumnID last.` ?